modules/trusted-firmware-m/Kconfig.psa.defconfig

```diff
@@ -97,7 +97,7 @@ config PSA_WANT_ALG_OFB
 	default y
 config PSA_WANT_ALG_XTS
 	bool
-	default y
+	#default y -- Currently not supported
```

Comment:
this doesn't remove the setting, only changing its default.

Other part can still do a `select PSA_WANT_ALG_XTS`.
I do notice it's not use configurable, as it is promptless.

Secondly, we should not have code commented out like this, better to remove, a new commit can always re-introduce the default later.

Reply:
This is a defconfig, the real configuration also has a removed prompt. The problem was that the defconfig made this a valid configuration (when it currently isn't supported)

We will rectify this missing feature in upcoming work (after nRF Connect SDK 1.9.0 release)

Reply:
so then reintroduce the setting in the defconfig at that time.

I would prefer not to have a setting doing nothing in this file.

Reply:
This file contains a one to one match of all  available configurations in the order that they are presented in mbed TLS PSA configuration. We are going for completeness to not hide away missing features, so I disagree on this

Reply:
What about a comment that says

# PSA_WANT_ALG_XTS is currently not supported.


In any case, we can't have commented code like this

Reply:
Accepted. Will rework the whole file for all items like this!

Reply:
```suggestion
	# PSA_WANT_ALG_XTS is currently not supported.
```

Reply:
Removed the config and replaced it with a comment. But I'm not pushing the fix because we don't want to cancel the CI run.